keyboards/clueboard/66/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+# rules.mk overrides to enable VIA
+
+VIA_ENABLE = yes
+MOUSEKEY_ENABLE = no
```

Comment:
I know via recommends this, but we should have enough space that this can stay enabled. I'd prefer to leave it on if possible.

```suggestion
```

Reply:
It's too large for Rev3 with MouseKey enabled

```
Checking file size of clueboard_66_rev3_via.hex                                                                                                                                                                                                                                   
 * The firmware is too large! 30288/28672 (1616 bytes over) 
```

Reply:
I was able to re-enable mousekeys by enabling `LINK_TIME_OPTIMIZATION_ENABLE`

Reply:
Also, I think the main reason for this recommendation is that VIA currently does not have the mousekey keycodes.